test/v1/fixtures/SubmitUKPropertyBsasRequestBodyFixtures.scala

```diff
@@ -324,4 +341,137 @@ object SubmitUKPropertyBsasRequestBodyFixtures {
       )
     )
   }
+
+  val desResponse: String => String = (bsasId: String) =>
```

Comment:
might be worth adding 'fhl' to the val name to make it clear what type of property it belongs to.